examples/problems/1000_compute_feature_correlation.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>add a description what you are doing here. Didn't we send you what we did for the paper? i.e. the correlation for the push-forward distribution?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>